### DISCUSSION TITLE
keyball39のトラックボールのセンサとレンズ

### DISCUSSION THREAD

**Discussion by tattomo3** [Yowkees/keyball#593] [Q&A]
keyball39を使用しているのですが、トラックボール部分のセンサ(PMW3360DM-T2QUだったと思います・・・)とレンズ(プラスチックの部分)を破損させてしまいました。
取り扱っているサイトが見つけられなかったため、その2点の部品だけ購入させていただきたいと考えているのですが、可能でしょうか？
すみませんがよろしくお願いします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

白銀ラボのウェブショップからKeyballの購入者様へ向けて部品の販売を行っておりますので、ウェブショップにありますお問い合わせフォームからご連絡をお願いいたします。
